modules/oadp-backup-restore-for-large-usage.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// about-installing-oadp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-backup-restore-for-large-usage_{context}"]
+= NodeAgent CPU for large usage
+
+Testing shows that increasing `NodeAgent` CPU can significantly improve backup and restore times when using {oadp-first}.
+
+It is not recommended to use Kopia without limits in production environments on nodes running production workloads due to Kopia’s aggressive consumption of resources.
+
+However, running Kopia with limits that are too low results in CPU limiting and slow-performing backups and restore situations.
+
+The testing undertaken observed that running Kopia with 20 cores and 32 Gi memory supported backup and restore operations of over 100GB of data, multiple namespaces, or over 2000 pods in a single namespace.
+
+Testing detected no CPU limiting or memory saturation with these resource specifications.
+
+You can set these limits in Ceph MDS pods by following the procedure in https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html/troubleshooting_openshift_data_foundation/changing-resources-for-the-openshift-data-foundation-components_rhodf#changing_the_cpu_and_memory_resources_on_the_rook_ceph_pods[Changing the CPU and memory resources on the rook-ceph pods].
+
+You need to add the following lines to the storage cluster Custom Resource (CR):
```

Comment:
Maybe say what this config does like: 

`You need to add the following lines to the storage cluster Custom Resource (CR) <to add blank blank or what adding these parameters does> `